ci/citest/acceptance-test/multibox/10.0.100.14-vdc-executor-lxc/build.sh

```diff
@@ -16,6 +16,7 @@ IND_STEPS=(
     "hosts"
     "disable-firewalld"
     "epel"
+    "bridge-utils"
```

Comment:
Why is this step here? Bridge utils is required by the package so we shoudl rely on that to install it. Because master is red, we should merge asap so I'm not going to hold back the PR for this but it should be fixed. We should always let our packages resolve their dependencies automatically. Otherwise we will not notice when there's problems with them.